doc/integrator/extend_data_model.rst

```diff
@@ -244,5 +245,53 @@ And finally in ``geoportal/<package>_geoportal/__init__.py`` replace ``config.sc
 
     with PermissionSetter(config):
         # Scan view decorator for adding routes
-        config.scan('<package>.admin.views')
-    config.scan(ignore='<package>.admin.views')
+        config.scan('<package>_geoportal.admin.views')
+    config.scan(ignore='<package>_geoportal.admin.views')
+
+Build and run the application:
+
+.. prompt:: bash
+
+   ./build <args>
+   docker-compose up -d
+
+Get and run the SQL command to create the tables:
+
+Run Python console:
+
+.. prompt:: bash
+
+   docker-compose exec geoportal python3
+
+Execute the following code:
+
+.. code:: python
+
+   from c2c.template.config import config
+
+   config.init('config.yaml')
```

Comment:
May be `config.init('/etc/geomapfish/config.yaml')`, but you should now better than me

Reply:
+1 Thanks :-)